src/buildCommon.js

```diff
@@ -608,7 +609,7 @@ const makeVList = function(params: VListParam, options: Options): DomSpan {
             pstrutSize = Math.max(pstrutSize, elem.maxFontSize, elem.height);
         }
     }
-    pstrutSize += 2;
+    pstrutSize = utils.round(pstrutSize + 2);
```

Comment:
I'm glad to see you didn't need to add `utils.round` in too many places.

Reply:
```suggestion
    pstrutSize += 2;
```